### PR TITLE
Add Software Supply Chain Best Practices v2 markdown

### PR DESCRIPTION
**Pull request by mnm678**
The Software Supply Chain working group has been working on an update to the Software Supply Chain Best Practices Paper. This is the markdown version of the document we have been working on.

### CODE REVIEW

**Comment by brandtkeller** [comment]
@mnm678 To what extent do you want to resolve the linter findings before merging this? No hard opinions here - might be a few spelling items worth review.

File: community/working-groups/supply-chain-security/suply-chain-security-paper-v2/SSCBPv2.md
Comment (jkjell) [inline community/working-groups/supply-chain-security/suply-chain-security-paper-v2/SSCBPv2.md:6]: It looks like GitHub auto-generates this heading ids: https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#section-links.

**Comment by jkjell** [comment]
I made this PR for the linter config which should resolve those. I agree that this should be allowed. https://github.com/cncf/tag-security/pull/1397

**Review by brandtkeller — COMMENTED** [review]
Minor callouts - feel free to consider these against the source-of-truth and otherwise resolve. 

I will follow-up with resolving these comments to approve if helpful.

**Comment by mnm678** [comment]
Thanks @brandtkeller! Feedback applied

File: community/working-groups/supply-chain-security/suply-chain-security-paper-v2/SSCBPv2.md
Comment (brandtkeller) [inline community/working-groups/supply-chain-security/suply-chain-security-paper-v2/SSCBPv2.md:351]:
```suggestion
You may think that it’s the problem of those providing the software.
```

**Review by brandtkeller — CHANGES_REQUESTED** [review]

File: community/working-groups/supply-chain-security/suply-chain-security-paper-v2/SSCBPv2.md
Comment (brandtkeller) [reply community/working-groups/supply-chain-security/suply-chain-security-paper-v2/SSCBPv2.md:496]: @mnm678 If I am looking at this change correctly - this can be consumed by the **Verification** bullet point item - is that intended?

Comment (mnm678) [reply community/working-groups/supply-chain-security/suply-chain-security-paper-v2/SSCBPv2.md:496]: Oh, it looks like the * was removed somehow, fixing now

Comment (brandtkeller) [reply community/working-groups/supply-chain-security/suply-chain-security-paper-v2/SSCBPv2.md:496]: This is the only remaining conversation I have open to resolve before approval.

Comment (mnm678) [reply community/working-groups/supply-chain-security/suply-chain-security-paper-v2/SSCBPv2.md]: I missed that my push failed. Should be fixed now

**Review by brandtkeller — APPROVED** [review]
Content looks great - all of my concerns were resolved.

**Pull request merged**
